cumulus/parachains/runtimes/assets/asset-hub-rococo/src/weights/pallet_balances.rs

```diff
@@ -159,4 +159,12 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for WeightInfo<T> {
 			.saturating_add(Weight::from_parts(0, 1501))
 			.saturating_add(T::DbWeight::get().reads(1))
 	}
+	fn burn_allow_death() -> Weight {
+		// TODO
```

Comment:
```suggestion
		// FAIL-CI
```
So we dont accidentally merge it like this.